.github/workflows/release.yml

```diff
@@ -7,27 +7,36 @@ on:
         type: choice
         description: Select the version change
         required: true
-        default: 'minor'
+        default: "minor"
         options:
           - major
           - minor
           - patch
 
+permissions:
+  contents: read
+
 jobs:
   release:
     runs-on: ubuntu-latest
     env:
       VERSION_CHANGE: ${{ github.event.inputs.versionChange }}
+
+    permissions:
+      contents: write
+      id-token: write
```

Comment:
Oh, I forgot about this.
`id-token` is used for provenance.
`contents` is overly broad, but it's the only way to grant releases permissions, and it's the default if I hadn't set it otherwise up above.